src/wp-includes/class-wp-user-query.php

```diff
@@ -1145,6 +1168,9 @@ public function __set( $name, $value ) {
 	public function __isset( $name ) {
 		if ( in_array( $name, $this->compat_fields, true ) ) {
 			return isset( $this->$name );
+		} else {
```

Comment:
The `else` does not need to be explicitly declared here. The `return` in the `if` will bail out of the method, meaning the `else` code will not run. It's an "implied" `else` branch strategy.